CHANGELOG.md

```diff
@@ -69,6 +69,7 @@ Ref: https://keepachangelog.com/en/1.0.0/
 
 ### API Breaking Changes
 
+* (x/distribution) [#17657](https://github.com/cosmos/cosmos-sdk/pull/17657) The distribution keeper now takes 2 more arguments, msg service router and grpc query router.
```

Comment:
Can we get a state machine breaking changelog (migration from community pool from distribution to x/protocolpool) and a Feature changelog?